Makefile

```diff
@@ -68,7 +68,7 @@ assets:
 
 promu:
 	@GOOS=$(shell uname -s | tr A-Z a-z) \
-	GOARCH=$(subst x86_64,amd64,$(patsubst i%86,386,$(shell uname -m))) \
+	GOARCH=$(subst x86_64,amd64,$(patsubst i%86,386,$(patsubst arm%,arm,$(shell uname -m)))) \
```

Comment:
How about `arm64` ?

Unfortunately, I don't have any `ARM` stuff for testing.

Reply:
How about aarch64?